### ISSUE TITLE
Cyclic import with dynamic paths not detected

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#380]
An imported file imports itself. All imports based on dynamic paths.

Result:

The import is not resolved. The cyclic import is not detected.

```
{'[import]': '... /dynamic_imports/cyclic_import_2.jsonp',
 'imports_dir': 'dynamic_imports',
 'root_name': 'cyclic_import'}
```

Expected: Cyclic import detected

Detection must work on any level. Can be also that cyclic import appears two levels deeper.

**Comment by HolQue** [comment]
Imports are resolved now. But: https://github.com/test-fullautomation/python-jsonpreprocessor/issues/387

**Comment by namsonx** [comment]
Hello Holger,

This ticket was solved on stabi branch, could you please verify it again.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
merged for version 0.13.1
